controllers/release/adapter.go

```diff
@@ -761,20 +767,27 @@ func (a *adapter) createManagedCollectorsPipelineRun(releasePlanAdmission *v1alp
 
 // createTenantCollectorsPipelineRun creates a PipelineRun to run the collectors Pipeline for collectors in the ReleasePlan.
 func (a *adapter) createTenantCollectorsPipelineRun(releasePlan *v1alpha1.ReleasePlan, releasePlanAdmission *v1alpha1.ReleasePlanAdmission) (*tektonv1.PipelineRun, error) {
+	var url string
 	var revision string
 	var err error
+	var err2 error
 
 	if releasePlanAdmission != nil {
-		revision, err = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetRevision()
+		url, err = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetUrl()
+		revision, err2 = releasePlanAdmission.Spec.Pipeline.PipelineRef.GetRevision()
```

Comment:
This looks ugly (err2). Just move it after the if block and reuse the same error.

Reply:
fixed.